packages/ui-components/src/components/date-time-input/test/date-time-input.e2e.ts

```diff
@@ -0,0 +1,61 @@
+import { E2EPage, newE2EPage, EventSpy, E2EElement } from '@stencil/core/testing';
+
+describe('Date Time Input (end-to-end)', () => {
+	let page: E2EPage;
+
+	describe('when renders with default props', () => {
+		beforeEach(async () => {
+			page = await newE2EPage();
+			await page.setContent('<kv-date-time-input></kv-date-time-input>');
+		});
+
+		describe('when user changes text', () => {
+			let spyChangeEvent: EventSpy;
+			let dateTimeInputComponent: E2EElement;
+
+			beforeEach(async () => {
+				dateTimeInputComponent = await page.find('kv-date-time-input');
+				spyChangeEvent = await dateTimeInputComponent.spyOnEvent('textChange');
+
+				const dateTimeInputElement = await page.find('input');
+				await dateTimeInputElement.type('Teste');
+
+				await page.waitForTimeout(2);
```

Comment:
This is deprecated. We should not wait for a specific period of time.
Instead, we should wait for an event to be dispatched. See if this works:

```typescript
await page.waitForEvent('textChange');
```

Reply:
It did not work, but waitForChanges did!